server/src/core/server/graph/schema/schema.graphql

```diff
@@ -4459,6 +4480,120 @@ type Queues {
   unarchiver: Queue!
 }
 
+################################################################################
+## Notifications
+################################################################################
+
+type Notification {
+  """
+  id is the uuid identifier for the notification.
+  """
+  id: ID!
+
+  """
+  ownerID is the string identifier for who this notification is directed to.
+  """
+  ownerID: ID!
+
+  """
+  createdAt is when this notification was created, used for sorting.
+  """
+  createdAt: Time!
+
+  """
+  title is the title text of this notification.
+  """
+  title: String
+
+  """
+  body is the text content of this notification.
+  """
+  body: String
+
+  """
+  comment is the optional comment that is linked to this notification.
+  """
+  comment: Comment
+
+  """
+  dsaReport is the details of the DSA Report related to the notification.
+  This is usually in reference to the comment that is also related to
+  the notification.
+  """
+  dsaReport: NotificationDSAReportDetails
+}
+
+type NotificationEdge {
+  """
+  node is the Flag for this edge.
+  """
+  node: Notification!
+
+  """
+  cursor is used in pagination.
+  """
+  cursor: Cursor!
+}
+
+type NotificationsConnection {
+  """
+  edges are a subset of FlagEdge's.
+  """
+  edges: [NotificationEdge!]!
+
+  """
+  nodes is a list of Flags.
+  """
+  nodes: [Notification!]!
+
+  """
+  pageInfo is information to aid in pagination.
+  """
+  pageInfo: PageInfo!
+}
+
+type NotificationDSAReportDetails {
+  """
+  id is the primary identifier of the DSA report.
+  """
+  id: ID!
+
+  """
+  publicID is the public identifier that is human readable
+  for the DSA report.
+  """
+  publicID: String!
```

Comment:
fyi, I updated this to use `referenceID` instead of `publicID` here

Reply:
Yup, I noticed this, updated and fixed!